Makefile

```diff
@@ -89,6 +89,16 @@ generate-manifests: ## Generate manifests using Helm.
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set service.annotations.'service\.beta\.kubernetes\.io\/aws-load-balancer-type'="nlb" -n nginx-gateway -s templates/service.yaml > $(strip $(MANIFEST_DIR))/service/loadbalancer-aws-nlb.yaml
 	helm template nginx-gateway $(CHART_DIR) $(HELM_TEMPLATE_COMMON_ARGS) --set service.type=NodePort --set service.externalTrafficPolicy="" -n nginx-gateway -s templates/service.yaml > $(strip $(MANIFEST_DIR))/service/nodeport.yaml
 
+.PHONY: generate-manifests-plus
+generate-manifests-plus: ## Generate manifests using Helm for NGINX Plus.
```

Comment:
I honestly think I'd prefer this to be under `generate-manifests`, so we can just create everything in one command. Be sure to actually commit the generated files too.

For the plus image name, NIC just generates theirs as a placeholder `nginx-plus-ingress` and then tells users in the docs to ensure they update the image to the correct Plus image when installing. We should do this too, since we can't include the actual Plus registry as that image path (since it's private).

We can have two directories of manifests. One for OSS version, one for plus.

Reply:
Why do we need two directories of manifests? The CRDs and Services are the same for both OSS and plus.

Reply:
I'm open to a better way of organizing. I just figured each option could be fully self-contained.

Reply:
Yeah, I don't see the need of having a separate target at all actually. Can't we simply add this new helm command to the existing generate-manifests target and have two deployment manifests with different names?